web/src/auth/LoginPage.js

```diff
@@ -50,6 +51,7 @@ class LoginPage extends React.Component {
       validPhone: false,
       loginMethod: "password",
     };
+    this.captchaModal = React.createRef();
```

Comment:
Can not login in.

![image](https://user-images.githubusercontent.com/71440988/196415733-34dc585e-9a78-4e27-a4eb-441cad8b1064.png)